README.md

```diff
@@ -24,17 +26,29 @@ render() {
 }
 ```
 
+Or import icons individually:
+
+```
+import GridiconAddImage from 'gridicons/cjs/add-image';
+//...
+render() {
+    return <GridiconAddImage />;
+}
```

Comment:
I think you can drop the `render()` {}` example here, it doesn't really make much sense outside the context of a component. I think it's fine to simply show what the `import` will look like here.

Reply:
I prefer to keep it: it instructs the consumer of the API how to use the component noting that in the general case you'll use the icon prop and if you import the individual component you don't. That's pretty clear to my eyes, but I'd rather err on the side of being obvious for new users.

Reply:
What about making it a functional component to have a more viable/standalone example? 😁

Reply:
E.g.

```js
import GridiconAddImage from 'gridicons/cjs/add-image';
//...
function MyComponent() {
    return <GridiconAddImage />;
}

```

Reply:
I've had too much npm for today :sweat_smile:

Reply:
I think the render() is more accessible for folks who have just started with React, let's keep in it. :)